src/model/job-advert.v1.yaml

```diff
@@ -0,0 +1,45 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Job advert
+type: object
+allOf:
+  - $ref: ../snippets/job-advert.v1.yaml
+  - properties:
+        description:
+            description: Description
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        responsibilities:
```

Comment:
I think we should unify `description`, `responsibilities` and `experience` into the generic `content` mentioned, perhaps leave `termsAndConditions` as a separate element as it seems important/legal?